examples/plot_nii_to_graph.py

```diff
@@ -0,0 +1,201 @@
+"""
+.. _nii_to_graph:
+
+=========================================================
+Compute connecivity matrices and graph properties from nii files
+=========================================================
```

Comment:
Nice, I didn't know you had started using sphinx gallery. That is great.

Reply:
;)